web/ITechArt.StudentLabs/ITechArt.StudentsLab.BusinessLayer/Services/FeedbackService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using ITechArt.StudentsLab.BusinessLayer.Contracts;
+using ITechArt.StudentsLab.DataAccessLayer.Contracts;
+using ITechArt.StudentsLab.BusinessLayer.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ITechArt.StudentsLab.DataAccessLayer.Models;
+using Mapster;
+
+namespace ITechArt.StudentsLab.BusinessLayer.Services
+{
+    public class FeedbackService: IFeedbackService
+    {
+        private readonly IFeedbackRepository _feedbackRepository;
+
```

Comment:
If you use two line to separate constructor from other code then one more line should be added after constructor. In other case here should be one line